### PR TITLE
Adds Nanotrasen balloon

### PR DESCRIPTION
**Pull request by Bawhoppen**
Glory to Nanotrasen.

Can be ordered from cargo for 200 points.

Also git is tough, don't yell at me

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -101,6 +101,18 @@
...
+	name = "nanotrasen balloon"
+	desc = "There is a tag on the back that reads \"FUK SYNDIES!11!\"."
+	throwforce = 0
+	throw_speed = 3
+	throw_range = 7
+	force = 0
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "ntballoon"
+	item_state = "ntballoon"
+	w_class = 4
```
Comment (duncathan) [inline code/game/objects/items/toys.dm]: You can get rid of everything here except name, desc, icon_state and item_state. The rest is inherited.

Comment (Bawhoppen) [inline code/game/objects/items/toys.dm]: I'm not a coder so I just copypasted everything but ok

**Comment by GunHog** [comment]
Adding a new item without posting pics? Shame!!!

**Comment by Bawhoppen** [comment]
its a fucking syndie balloon thats blue and is NT instead of an S

I intended to post a picture but git is tough :(

ss 2015-10-23 at 01 51 35

**Comment by Lumipharon** [comment]
You should use the N logo that we've got on the homepage/other places, rather then NT.

**Comment by RemieRichards** [comment]
make it the proper N logo, with the little triangles in the top left and bottom right.

**Comment by KorPhaeron** [comment]
I get that its supposed to be a joke but 200 points is really a bit absurd.

Gonna be like the hat crate in that people just pray for it instead of buying it

**Comment by Bawhoppen** [comment]
I just wanted it to be an opposite of the Syndie balloon

Also pride is expensive

**Comment by Shapsy** [comment]
That's a very square balloon

**Comment by Bawhoppen** [comment]
So is the syndie balloon

**Comment by Fox-McCloud** [comment]
Shouldn't this be like 20 cargo points? At 200 it's never going to be ordered after it's been in the game for like 2 weeks.

**Comment by Bawhoppen** [comment]
@Fox-McCloud Its not like its functional, its supposed to be a novelty just like the hat crate, for if you want to express your Nanotrasenalism.

**Comment by Razharas** [comment]
>Not using the standard logo and using some letters instead

Wowowow
Please apply yourself

**Comment by Bawhoppen** [comment]
@Razharas https://www.youtube.com/watch?v=M_azCIe_0Kk

**Comment by Fox-McCloud** [comment]
@Bawhoppen 

You also get 3 hats. At the very least make it something less ridiculous like 50-60.

**Comment by Lumipharon** [comment]
You should be able to use a balloon on yourself to speak with a squeeky voice temporarily.
Or use it on itself to untie the balloon and make it fart around the room.

Why aren't these things, damn it.

**Comment by Bawhoppen** [comment]
@Fox-McCloud It's a novelty. It doesn't need to be easily acquirable.

**Comment by Dorsisdwarf** [comment]
@Bawhoppen fix yer PR or close yer PR

**Comment by Bawhoppen** [comment]
I didn't even remember this existed,

anyways raz is too personally insulted to ever let anything I pr get merged apparently

**Pull request closed without merging**